src/azure/pci-dss-3.2.1/rules/pci-dss-3.2.1-monitoring-check-3.ts

```diff
@@ -0,0 +1,73 @@
+export default {
+  id: 'azure-pci-dss-3.2.1-monitoring-check-3',  
+  title: 'Monitoring Check 3: Security Center default policy setting ‘Monitor Endpoint Protection’ should be enabled',
+  
+  description: 'When this setting is enabled, it recommends endpoint protection be provisioned for all Windows virtual machines to help identify and remove viruses, spyware, and other malicious software.',
+  
+  audit: '',
+  
+  rationale: '',
+  
+  remediation: `**Azure Portal**
+  
+  - Navigate to Azure Policy.
+  - Select the subscription and click Edit assignment.
+  - Select Parameters.
+  - In Monitor missing Endpoint Protection in Azure Security Center, select AuditIfNotExists.
+  - Click Review + save > save.
+  
+  **Azure CLI**
+  
+  - Remediation is not possible via the CLI.`,
+  
+  references: [
+      'https://docs.microsoft.com/en-us/azure/security-center/tutorial-security-policy',
+      'https://docs.microsoft.com/en-us/azure/security-center/security-center-policy-definitions',
+  ],
+  gql: `{
+    queryazurePolicyAssignment {
+      id
+      __typename
+      displayName
+      parameters {
+        key
+        value {
+          key
+          value
+        }
+      }
+    }
+  }`,
+  resource: 'queryazurePolicyAssignment[*]',
+  severity: 'medium',
+  conditions: {
+    or: [
+      {
+        path: '@.displayName',
+        notEqual: 'Monitor missing Endpoint Protection in Azure Security Center'
+      },
+      {
+        path: '@.parameters',
+        array_any: {
+          and: [
+            {
+              path: '[*].key',
+              equal: 'effect',
+            },
+            {
+              and: [
+                {
+                  path: '[*].value',
```

Comment:
Yes, because that `value` property is an array of key-value pairs

![image](https://user-images.githubusercontent.com/64504276/176000379-8b211486-2f75-49b8-bf56-c09745354a87.png)